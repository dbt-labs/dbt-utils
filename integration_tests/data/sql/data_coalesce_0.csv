field_1,field_2,field_3
0,0,0
,1,1
2,,2
3,3,
