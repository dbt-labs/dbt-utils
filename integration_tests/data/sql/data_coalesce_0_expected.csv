field_1,field_2,field_3
0,0,0
0,1,1
2,0,2
3,3,0
